src/tokenserver/db/mod.rs

```diff
@@ -1,5 +1,8 @@
 pub mod mock;
 pub mod models;
-mod params;
+pub mod params;
 pub mod pool;
 pub mod results;
+
+pub const SYNC_1_1_SERVICE_ID: i32 = 1;
```

Comment:
I simplified the logic from the legacy Tokenserver by hard-coding the only two services that will ever exist with static DB IDs. This simplifies some of the database querying.